cpp/src/jit/cache.cpp

```diff
@@ -14,41 +14,50 @@
  * limitations under the License.
  */
 
+#include <boost/filesystem.hpp>
```

Comment:
Does this introduce a new dependency on boost in libudf?!

Reply:
I think `boost::filesystem` is an existing dependency because of Arrow: https://github.com/rapidsai/cudf/blob/f1650d664fa76773cf63f78570a2a94c43fed40b/cpp/CMakeLists.txt#L184-L192

Reply:
Yes it was included because of Arrow, but if we're explicitly depending on Boost we should make sure to note it somewhere.